release-notes/con-release-notes-overview.adoc

```diff
@@ -17,13 +17,19 @@ Benefits of {product} include:
 * *Robust enterprise Role-Based Access Control (RBAC)*: Empowers administrators to create roles, assign users or groups to roles, and implement robust security policies for enhanced access control.
 
 == Supported platforms
-* OpenShift Container Platform 4.12, 4.13, 4.14, 4.15
+* OpenShift Container Platform 4.12, 4.13, 4.14, 4.15 
 * Red Hat OpenShift Service on AWS (ROSA)
 * Azure Red Hat OpenShift (ARO)
 * Amazon Elastic Kubernetes Service (EKS)
 * Microsoft Azure Kubernetes Service (AKS)
 * [Technical Preview] Google Kubernetes Engine (GKE)
 
+== Supported architectures
+
+Containers are available for the following CPU architectures:
+
+* AMD64 and Intel 64 (x86_64)
+
```

Comment:
I trust that you know that this is true and there are no errors here, but another SME or QE should review/confirm this before merging.

Reply:
Well, you can look up the options from https://catalog.redhat.com/software/containers/rhdh/rhdh-hub-rhel9/645bd4c15c00598369c31aba?architecture=amd64&image=660ebff0036e65b1bc611db4 if you don't believe me :rofl: 

![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/227597/724468d7-1790-459a-b218-618bc9864abe)

Reply:
this is how we documented arch support for Dev Spaces, which was available on 3 arches. https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.12/html-single/release_notes_and_known_issues/index#supported-platforms

Reply:
@nickboldt Cool, thanks. I didn't have the knowledge to confirm the technical accuracy of this change and wasn't sure how/where to check for confirmation and didn't want to merge anything without a tech check.

Reply:
@nickboldt From now on, we need to get an SME or QE reviewer for all PRs merging to the docs repo. In the interests of time, if you could find someone to do the review, that would be a great help. Otherwise, the docs team can find someone.

Reply:
I thought I was an SME here, given I do the builds and cited my sources, go ahead and conscript someone in QE or another SME to click on the above links to verify my claim that we only build on one architecture. In this case it feels like you're making work for others, but I understand that process-wise it's uncool to have the PR come from the SME who does the SME review.

On the other hand, I've provided you with the doc PR that a writer would have otherwise had to submit to ME to review, so I feel like ultimately we're getting to the same place?

Reply:
@nickboldt I totally hear you. But as a docs expert, I am still required to have my work checked by another docs expert in the form of a peer review, even for minor typo fixes. This makes me grumble, but I accept that those are the rules of contributing to this repo. Maybe the rules themselves need to be revisited. But in the meantime, Brian is trying to standardize the review process for all docs repo contributions. Otherwise the docs team is held to rules that other contributors are exempt from, despite being the primary maintainers of the repo.